### DISCUSSION TITLE
Cornell box

### DISCUSSION THREAD

**Discussion by kironia** [eltikia/small-raytracer#6] [Q&A] [answered]
What's in the box?

**Comment by Stagyrite** [comment, marked as answer]
This raytracer produces an image that consists of four walls. The left one is red, and the right one is green. The ceiling, the floor and the remaining wall are white. The light source is white. Contrary to the Cornell box, it's not placed on the ceiling. What's more different is that there is not a single object there. One would expect 3 spheres rendered.

To better show what's there, I used Gimp Equalize and White Balance commands.
![equalized](https://github.com/eltikia/small-raytracer/assets/55529808/fa0c934a-cb40-4dda-a17b-f17a04b98fd4)
